### PR TITLE
feature(position): add RTL-friendly aliases -start and -end for -left and -right

### LINKED ISSUE #707: Add RTL support
- ~close button should be placed in top-left corner~ (done in #774)
- for the `position` parameter there not be `-start` and `-end` aliases for `-left` and `-right`

### PR DESCRIPTION
**Pull request by limonte**
Fixes #707 

145B larger than master. The total cost of the RTL support is less than 200B, I guess that's totally fine.

@toverux @acupajoe are you okay with making `start` and `end` default and `left` and `right` as RTL-unfriendly aliases?

### CODE REVIEW

File: sweetalert2.d.ts [hunk 1]
```diff
@@ -322,6 +322,9 @@ declare module 'sweetalert2' {
          * @default 'center'
          */
         position?:
+            'top' | 'top-start' | 'top-end' |
+            'center' | 'center-start' | 'center-end' |
+            'bottom' | 'bottom-start' | 'bottom-end' |
```
Comment (toverux) [inline sweetalert2.d.ts]: There are duplicates in this list of types. Is that desired?

**Comment by toverux** [comment]
> are you okay with making start and end default and left and right as RTL-unfriendly aliases?

Yes totally, the sooner the better, so user will have less efforts to make the transition when the left/right equivalents get deleted (if that's still the plan).

I'd even remove the `You can also use RTL-unfriendly `'left'` and `'right'` instead of `'start'` and `'end'`.` part in the documentation. Just replace with the new aliases, this won't be a breaking change since left/right variants will still work, and new users will not even know these have existed.

File: sweetalert2.d.ts [hunk 2]
```diff
@@ -322,6 +322,9 @@ declare module 'sweetalert2' {
...
             'top' | 'top-left' | 'top-right' |
             'center' | 'center-left' | 'center-right' |
             'bottom' | 'bottom-left' | 'bottom-right';
```
Comment (toverux) [inline sweetalert2.d.ts]: There are duplicates in this list of types. Is that desired (simmetry I assume, but catched my eye)?

**Comment by limonte** [comment]
> Yes totally, the sooner the better, so user will have less efforts to make the transition when the left/right equivalents get deleted (if that's still the plan).

Agree.

> I'd even remove the `You can also use RTL-unfriendly 'left' and 'right' instead of 'start' and 'end'.` part in the documentation. Just replace with the new aliases, this won't be a breaking change since left/right variants will still work, and new users will not even know these have existed.

I was thinking about that as well but then added the sentence about 'left' and 'right' just in case. But, yeah, that's wasn't necessary. Removed.

File: sweetalert2.d.ts (hunk 2 above)
Comment (limonte) [reply sweetalert2.d.ts]: Oops, removed, thanks for catching this.

**Comment by acupajoe** [comment]
Thanks for getting this one done @limonte!

**Pull request merged**